server/pulp/server/db/manage.py

```diff
@@ -191,6 +196,35 @@ def main():
         options = parse_args()
         _start_logging()
         connection.initialize(max_timeout=1)
+
+        active_workers = status.get_workers()
+        if active_workers:
+            last_worker_time = max([worker['last_heartbeat'] for worker
+                                    in active_workers])
+            time_from_last_worker = UTCDateTimeField().to_python(datetime.now()) - last_worker_time
+            wait_time = timedelta(seconds=constants.MIGRATION_WAIT_TIME) - time_from_last_worker
+
+            if wait_time > timedelta(0):
+                print _('The following processes might still be running:')
+                for worker in active_workers:
+                    print _('\t%s' % worker['name'])
+
+                for i in range(wait_time.seconds, 0, -1):
+                    print _('\rPlease wait %s seconds while Pulp confirms this.' % i),
+                    sys.stdout.flush()
+                    time.sleep(1)
+
+                still_active_workers = [worker for worker in status.get_workers() if
```

Comment:
You do not need to change this unless you want to, but FYI this is very similar to the usage of [filter()](https://docs.python.org/2/library/functions.html#filter).